.travis.yml

```diff
@@ -1,10 +1,15 @@
 sudo: false
 language: python
 python:
-    - "2.7.11"
-    - "3.5"
-    - "3.6"
+  - 2.7.11
+  - 3.5
+  - 3.6
```

Comment:
Can you confirm these still get parsed as strings? I remember there was some issue with it ending up as float values and breaking the config and so I added the quotes.

Reply:
Also I'd rather not change the indent because it clutters up the diff.